src/utils/openai.ts

```diff
@@ -106,9 +148,35 @@ export const generateCommitMessage = async (
 	locale: string,
 	diff: string,
 	completions: number,
+	useConventionalCommits: boolean,
 	proxy?: string,
 ) => {
-	const prompt = getPrompt(locale, diff);
+	const basePrompt = getBasePrompt(locale);
+
+	const commitMessageFormatPrompt = getCommitMessageFormatPrompt(
+		useConventionalCommits,
+	);
+
+	const conventionalCommitsExtraContext = useConventionalCommits
+		? getExtraContextForConventionalCommits()
+		: '';
+
+	const messages: ChatCompletionRequestMessage[] = [
+		{
+			role: 'system',
+			content: `${basePrompt}\n${commitMessageFormatPrompt}`,
+		},
+		{
+			role: 'assistant',
```

Comment:
FYI, assistant is the robot (as a way to provide conversation history) so the conventional commits rules shouldn't be here.

This should be merged with the `system` above.

Reply:
Alright, I thought assistant was something else. changed it 👍